packages/dds/tree/src/test/feature-libraries/forest-summary/incrementalSummaryBuilder.spec.ts

```diff
@@ -0,0 +1,538 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "node:assert";
```

Comment:
since you are using the strict assert APIs due to this import, no need to use the strict methods on it like strictEqual below: thats redundant: you can use the shorter names.

Reply:
Not sure what you mean by "shorter names".

Reply:
Oh, I get it. Updated them to not use the term strict.